web/lattice_main/styles_map.py

```diff
@@ -0,0 +1,28 @@
+import patches
+from qubit_state import ActivityType
+
+styles_map = {patches.PatchType.Qubit : "darkkhaki",
```

Comment:
This is a quick solution made up on the spot, it doesn't have to stay if you think it's ugly